cmd/tidb-dashboard/main.go

```diff
@@ -73,6 +73,7 @@ func NewCLIConfig() *DashboardCLIConfig {
 	flag.StringVar(&cfg.CoreConfig.PDEndPoint, "pd", cfg.CoreConfig.PDEndPoint, "PD endpoint address that Dashboard Server connects to")
 	flag.BoolVar(&cfg.CoreConfig.EnableTelemetry, "telemetry", cfg.CoreConfig.EnableTelemetry, "allow telemetry")
 	flag.BoolVar(&cfg.CoreConfig.EnableExperimental, "experimental", cfg.CoreConfig.EnableExperimental, "allow experimental features")
+	flag.BoolVar(&cfg.CoreConfig.AllowNonRootLogin, "allow-non-root-login", cfg.CoreConfig.AllowNonRootLogin, "allow non root sql user login")
```

Comment:
How about calling it EnableNonRootLogin to be unified with the rest?

Reply:
Done